src/mixins/controls.js

```diff
@@ -146,8 +147,48 @@ const OrbitControlsMixin = (superclass) => class extends superclass {
         this.renderer2 = this.getWebGLRenderer({alpha: true});
         this.renderer2.setClearColor("#FFFFFF", 0);
         this.renderer2.setSize(containerDimension, containerDimension);
+				this.renderer2.domElement.style.position = 'absolute';
+				this.renderer2.domElement.style.right = '10px';
         this.container.prepend(this.renderer2.domElement);
 
+        //For axis labels
+				this.labelRenderer = new CSS2DRenderer();
```

Comment:
`renderer2CSS2D` - might be a better name. I makes it clear that it is related to `renderer2` and is used to deal with the CSS2D context